packages/fxa-auth-server/lib/oauth/events.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('../../config').getProperties();
+const db = require('./db');
+const P = require('../promise');
+const logger = require('./logging')('events');
+const Sink = require('fxa-notifier-aws').Sink;
+const HEX_STRING = require('./validators').HEX_STRING;
+
+let fxaEvents;
```

Comment:
The file is here, but isn't wired up to anything yet.

Reply:
Oh, it should be possible to skip the whole SQS queue to update the OAuth DB now, shouldn't it? Whenever a user is removed, call the OAuth DB's removeUser, and whenever a password reset or change happens, call removePublicAndCanGrantTokens.

Reply:
Thanks! Yeah, I forgot that auth-server was the source of these events. I'll see if making that change is practical in this phase